prac_05/emails.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404 - Prac 5
+Program: Emails
+"""
+
+def main():
+    """Create dictionary of emails to names, and displays it."""
+    email_to_name = {}
+    email = input('Email: ')
+
+    while email != "":
```

Comment:
Instead of checking if email is not an empty string, you could use a while True loop with a break statement when email is empty. This can make it clearer when the loop is intended to stop.